src/code.sh

```
@@ -5,9 +5,20 @@
 set -e -x -o pipefail
 
 dx download "$input_bam" -o "$input_bam_name"
-dx download "$input_bam_index" -o "${input_bam_prefix}.bai"
+dx download "$input_bam_index" -o "$input_bam_index_name"
 dx download "$vcf_file" -o "$vcf_file_name"
 
+# if input_bam_name == cram, convert to bam and reindex
+if [[ $input_bam_name == *.cram ]]; then
+     echo "file is CRAM"
+
+     dx download "$reference_fasta" -o "$reference_fasta_name"
```

Shellcheck (0.10.0)

[warning] reference_fasta is referenced but not assigned.

(SC2154)